src/LumiCollimator_geo.cpp

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Justin Chan
+
+//==========================================================================
+//
+// Places a collimator along the beam line
+//
+//==========================================================================
+
+#include "DD4hep/DetFactoryHelper.h"
+#include <XML/Helper.h>
+
+using namespace std;
```

Comment:
Using namespace is generally to be avoided. Use specific using statements instead, e.g. `using dd4hep::DetElement`. If you end up needing more than three or so specific using statements, then using namespace is probably ok. Using namespace std can often be avoided. Applies to other files too.